snapcraft/internal/errors.py

```diff
@@ -138,6 +138,8 @@ class MissingCommandError(SnapcraftError):
     )
 
     def __init__(self, required_commands):
+        if isinstance(required_commands, list):
+            required_commands = required_commands.join(' ')
```

Comment:
It looks like you're no longer calling this exception with a list. Is this logic still required?